### PR TITLE
#166840996 Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by czarjulius**
#### What does this PR do?
Setup the travisCI with the readme badge

#### Description of Task to be completed?

- Create the .travis.yml file

- Add the readme badge to readme file

- Integrate the repository to the travisCI.org

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
<a href="https://www.pivotaltracker.com/story/show/166840996">166840996</a>

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

**Comment by codeBlock-1984** [comment]
Looks great. However, it seems you forgot to add a new line at the end of your .travis.yml file. Nice job Julius.

**Review by codeBlock-1984 — COMMENTED** [review]
Looks great. However, it seems you forgot to add a new line at the end of your .travis.yml file. Once you rectify that, I believe you are good to go. Nice job Julius.

**Review by Cavdy — CHANGES_REQUESTED** [review]
The setup is ok but I noticed you didn't specify the version of node js instead you wrote node... Please correct it... Here is a link to Travis [documentation](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/) on Nodejs

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+ - 'node'
```
Comment (phembarl) [inline .travis.yml]: Add an extra line. Otherwise, LGTM

**Review by chokonaira — COMMENTED** [review]
good to go

**Review by Cavdy — COMMENTED** [review]
Looks good to me...

**Review by Cavdy — APPROVED** [review]
Looks good to me...

**Review by codeBlock-1984 — APPROVED** [review]

**Review by chokonaira — APPROVED** [review]

**Pull request merged**